docs/resources/configuration_variable.sh

```diff
@@ -0,0 +1 @@
+terraform import env0_configuration_variable.my_config '{  "Scope": "PROJECT",  "ScopeId": "project id",  "name": "configuration variable name"}'
```

Comment:
this should be under the `examples` folder

Reply:
moved 7ab7d3f6807cd035167258b3c0f37cb30dfa4110
renamed 7ab7d3f6807cd035167258b3c0f37cb30dfa4110